ebos/eclproblem.hh

```diff
@@ -1589,17 +1593,14 @@ public:
             unsigned globalDofIdx = context.globalSpaceIndex(interiorDofIdx, timeIdx);
             unsigned pvtRegionIdx = pvtRegionIndex(context, spaceIdx, timeIdx);
             FaceDir::DirEnum dir = FaceDir::FromIntersectionIndex(indexInInside);
-            const auto& dirichlet = dirichlet_(dir)[globalDofIdx];
-            if (freebc_(dir)[globalDofIdx])
-                values.setFreeFlow(context, spaceIdx, timeIdx, boundaryFluidState(globalDofIdx, indexInInside));
-            else if (thermalbc_(dir)[globalDofIdx])
+            const auto& schedule = this->simulator().vanguard().schedule();
```

Comment:
This `schedule` object [isn't used anywhere](https://ci.opm-project.org/job/opm-common-PR-builder/5684/gcc/folder.-288973067/).  Is that intentional?